guides/common/modules/con_host-group-settings-and-nested-host-groups.adoc

```diff
@@ -0,0 +1,111 @@
+[id="Host_Group_Settings_and_Nested_Host_Groups_{context}"]
+= Host Group Settings and Nested Host Groups
+
+A host group can define many settings for hosts, such as Lifecycle Environment, Content View, or Ansible roles that are available to the hosts.
+
+NOTE: When you change the properties of a host group, the new settings do not propagate to the hosts assigned to the host group.
```

Comment:
This is important enough behavior that I think it should be elevated beyond a note (I almost skipped over it).

Reply:
Let's go with IMPORTANT, then. Let me know if this doesn't work for you.

Reply:
Works for me :+1: